tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestDisplayAndInputProcessor.java

```diff
@@ -477,6 +481,64 @@ public static PathSpaceData[] getArray() {
+        class PathSpaceAdjustData {
+            String locale;
+            String xpath;
+            String rawValue;
+            String normValue;
+
+            PathSpaceAdjustData(String loc, String path, String raw, String norm) {
+                this.locale = loc;
+                this.xpath = path;
+                this.rawValue = raw;
+                this.normValue = norm;
+            }
+        }
+
+        PathSpaceAdjustData[] testItems = {
+            new PathSpaceAdjustData("en",
+                    "//ldml/dates/calendars/calendar[@type=\"gregorian\"]/timeFormats/timeFormatLength[@type=\"short\"]/timeFormat[@type=\"standard\"]/pattern[@type=\"standard\"]",
+                    "h:mm a", "h:mm a"), // \u202F
+            new PathSpaceAdjustData("ja",
+                    "//ldml/dates/calendars/calendar[@type=\"gregorian\"]/dateTimeFormats/availableFormats/dateFormatItem[@id=\"hm\"]",
+                    "a K:mm", "a K:mm"), // \u202F
+            new PathSpaceAdjustData("en",
+                    "//ldml/dates/calendars/calendar[@type=\"gregorian\"]/dateTimeFormats/intervalFormats/intervalFormatItem[@id=\"hm\"]/greatestDifference[@id=\"a\"]",
+                    "h:mm - h:m a", "h:mm – h:m a"), // \u2009\u2013\u2009, \u202F
+            new PathSpaceAdjustData("uk",
+                    "//ldml/dates/calendars/calendar[@type=\"gregorian\"]/dateFormats/dateFormatLength[@type=\"medium\"]/dateFormat[@type=\"standard\"]/pattern[@type=\"standard\"]",
+                    "d MMM y 'р'.", "d MMM y 'р'."), // \u202F after y
+            new PathSpaceAdjustData("uk",
+                    "//ldml/dates/calendars/calendar[@type=\"gregorian\"]/dateTimeFormats/availableFormats/dateFormatItem[@id=\"yMMMd\"]",
+                    "d MMM y 'р'.", "d MMM y 'р'."), // \u202F after y
+            new PathSpaceAdjustData("uk",
+                    "//ldml/dates/calendars/calendar[@type=\"gregorian\"]/dateTimeFormats/intervalFormats/intervalFormatItem[@id=\"yMMMd\"]/greatestDifference[@id=\"M\"]",
+                    "d MMM - d MMM y 'р'.", "d MMM – d MMM y 'р'."), // \u2013, \u202F after y
+            new PathSpaceAdjustData("en",
+                    "//ldml/units/unitLength[@type=\"narrow\"]unit[@type=\"mass-gram\"]/unitPattern[@count=\"other\"]",
+                    "{0} g", "{0} g"), //  \u202F
+            new PathSpaceAdjustData("en",
+                    "//ldml/units/unitLength[@type=\"narrow\"]unit[@type=\"mass-gram\"]/unitPattern[@count=\"other\"]",
+                    "g {0}", "g {0}"), //  \u202F
+            new PathSpaceAdjustData("en",
+                    "//ldml/units/unitLength[@type=\"short\"]unit[@type=\"mass-gram\"]/unitPattern[@count=\"other\"]",
+                    "{0} g", "{0} g"), // \u00A0
+            new PathSpaceAdjustData("en",
+                    "//ldml/units/unitLength[@type=\"short\"]unit[@type=\"mass-gram\"]/unitPattern[@count=\"other\"]",
+                    "g {0}", "g {0}"), // \u00A0
```

Comment:
the comments on lines 529 and 532 refer to \u00A0, which, however, doesn't seem to occur in the data on those lines

Reply:
sorry, never mind, they do have \u00A0 but somehow it was changed to \u0020 when copy-pasting from browser